library/auth.inc

```diff
@@ -112,6 +113,16 @@ if (!isset($_SESSION["last_update"])) {
     }
 }
 
+
+
+// For RTL languages, substitute the same theme converted for RTL// 
+if ( $_SESSION['language_direction'] == 'rtl' && !strpos($GLOBALS['css_header'], 'rtl')) {
+    $GLOBALS['css_header'] = $css_header = str_replace( 'themes/style_', 'themes/rtl_style_', $GLOBALS['css_header'] );
+}
```

Comment:
Note that auth.inc script is not always called. Would it make sense to instead set $GLOBALS['css_header'] in the interface/globals.php script that is always called:
https://github.com/matrix-olegsv/openemr/blob/11082dc57af94ccc750936dcb983a9b851721d02/interface/globals.php#L215-L217